charts/opentelemetry-operator/templates/servicemonitor.yaml

```diff
@@ -24,12 +24,12 @@ spec:
   endpoints:
   {{- toYaml .Values.manager.serviceMonitor.metricsEndpoints | nindent 2 }}
   {{- with .Values.manager.serviceMonitor.relabelings }}
-  relabelings:
-  {{- toYaml . | nindent 4 }}
+    relabelings:
+  {{- toYaml . | nindent 6 }}
   {{- end }}
   {{- with .Values.manager.serviceMonitor.metricRelabelings }}
-  metricRelabelings:
-  {{- toYaml . | nindent 4 }}
+    metricRelabelings:
+  {{- toYaml . | nindent 6 }}
   {{- end }}
```

Comment:
suggestion: can we have all the blocks indentation match here?

Reply:
good call. just pushed those changes up